tests/drivers/adc/adc_dma/testcase.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+  tags: adc dma trigger
+tests:
+  drivers.adc-dma:
+    depends_on: adc && dma && pit
```

Comment:
Not sure if this syntax works. I think it should be `depends_on: adc dma pit`

Reply:
right, it is a set now.

Reply:
done